Content.Server/DeviceLinking/Systems/DoorSignalControlSystem.cs

```diff
@@ -65,15 +65,10 @@ private void OnSignalReceived(EntityUid uid, DoorSignalControlComponent componen
             }
             else if (args.Port == component.InBolt)
             {
-                if (state == SignalState.High)
+                var bolt = state == SignalState.High;
+                if (TryComp<AirlockComponent>(uid, out var airlockComponent))
                 {
-                    if(TryComp<AirlockComponent>(uid, out var airlockComponent))
-                        _airlockSystem.SetBoltsWithAudio(uid, airlockComponent, true);
-                }
-                else
-                {
-                    if(TryComp<AirlockComponent>(uid, out var airlockComponent))
-                        _airlockSystem.SetBoltsWithAudio(uid, airlockComponent, false);
+                    _airlockSystem.SetBoltsWithAudio(uid, airlockComponent, bolt);
```

Comment:
This breaks toggeling bolts with a momentary signal again. It might be worth checking if the signal is momentary and toggeling the bolt before handlign high and low signals.

Reply:
outdated since i merged your change and improved, but ill test it in a bit to double check

Reply:
yep works fine with momentary and high/low